packages/blockchain-wallet-v4-frontend/src/components/Flyout/Brokerage/OrderSummary.tsx

```diff
@@ -48,10 +58,23 @@ const BottomPromo = styled.div`
   justify-content: flex-end;
   flex-direction: column;
   height: 180px;
-  margin-bottom: 15px;
+  margin-bottom: 1rem;
+`
+
+const SecondaryInfoText = styled.div`
+  margin-top: 1.5rem;
```

Comment:
Possible to just use the `<Padding />` component here instead of creating new styled components just to add margin?

Reply:
I'll try, thanks dylan

Reply:
Hey @blockdylanb, I think that the component is using `px` as unit, can it be?
<img width="861" alt="image" src="https://user-images.githubusercontent.com/97299628/175100355-d51fc442-9a9c-46c6-b9d6-6d1b3db43068.png">

<img width="861" alt="image" src="https://user-images.githubusercontent.com/97299628/175100507-35337398-c5cb-4e40-9d85-54bd13bf986c.png">